src/server/server.ts

```diff
@@ -9,12 +9,14 @@ function getLedgerVersion(this: RippleAPI): Promise<number> {
   return this.connection.getLedgerVersion()
 }
 
-function connect(this: RippleAPI): Promise<void> {
+async function connect(this: RippleAPI): Promise<void> {
   return this.connection.connect()
 }
 
-function disconnect(this: RippleAPI): Promise<void> {
-  return this.connection.disconnect()
+async function disconnect(this: RippleAPI): Promise<void> {
+  // backwards compatibility: connection.disconnect() can return a number, but
+  // this method returns nothing. SO we await but don't return any result.
+  await this.connection.disconnect()
```

Comment:
This looks funky to me but I'm not a JS expert like you :)

implicitly this is adding a new statement that says:
```
return new Promise<Void>(function(res, rej) { res() });
```

As a non-JS programmer, I'd prefer you just make this explicit. If not, why not just return the number itself?

Reply:
We can make this change later 👍